pan-domain-node/src/panda.ts

```diff
@@ -57,6 +57,8 @@ export function verifyUser(pandaCookie: string | undefined, publicKey: string, c
         return { status: AuthenticationStatus.INVALID_COOKIE };
     }
 
+    const currentTimestamp = currentTime.getTime();
```

Comment:
Could you add seconds/milliseconds to the variable name to help reduce the kind of confusion that led to this mistake?

Reply:
```suggestion
    const currentTimestampInMilliseconds = currentTime.getTime();
```

Reply:
(Oh, although I know see that the problem is not to do with seconds vs milliseconds, it's about getting an epoch time vs the milliseconds since the last second. Still, can't hurt to be explicit)